docs/sql-guide/functions/function-str.md

```diff
@@ -0,0 +1,88 @@
+---
+title: STR()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# STR() function
+
+`STR()` function returns the given number formatted to a String of a certain length, with given precision.
+
+## Syntax
+
+```
+STR(num_expr, lenofstr, precision)
```

Comment:
As above with the way of consistently describing the original value.

This might be:

* `int_value`
* `value`
or whatever else.

of course, if it really **is** an expression then that would need to be clarified better.

Reply:
`lenofstr` could be changed to `string_length` so it's easier to read?